src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilterTest.java

```diff
@@ -506,7 +651,7 @@ public void testFilterWithIdAndCustomMessageFormat() throws Exception {
             createModuleConfig(SuppressWithPlainTextCommentFilter.class);
         filterCfg.addAttribute("offCommentFormat", "CHECKSTYLE stop (\\w+) (\\w+)");
         filterCfg.addAttribute("onCommentFormat", "CHECKSTYLE resume (\\w+) (\\w+)");
-        filterCfg.addAttribute("checkFormat", "$1");
+        filterCfg.addAttribute("idFormat", "$1");
```

Comment:
changed `checkFormat` to `idFormat` because test is `testFilterWithIdAndCustomeMessageFormat` and was using `checkFormat` to match by ID

Reply:
>I can add additional tests like 

Please add, the more tests the better

Reply:
Added `testFilterWithCheckAndCustomMessageFormat`